config/initializers/filter_parameter_logging.rb

```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
+
+# Configure sensitive parameters which will be filtered from the log file.
+Rails.application.config.filter_parameters += [:password]
```

Comment:
I think this can be removed from `application.rb` now.

Reply:
Yup, done.